src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonUtilities.Schema.cs

```diff
@@ -199,18 +199,31 @@ private static JsonNode CreateJsonSchemaCore(
                 (schemaObj = [])[SchemaPropertyName] = SchemaKeywordUri;
             }
 
-            if (description is not null)
+            if (hasDefaultValue)
             {
-                (schemaObj ??= [])[DescriptionPropertyName] = description;
+                if (inferenceOptions.RequireAllProperties)
+                {
+                    // Do not emit default values for schemas where all properties
```

Comment:
Can we add a comment about why we're applying this heuristic?

Reply:
The implied message is that `default` only makes sense for optional parameters, but of course that doesn't explain why we still embed it in the description.